### PR TITLE
[MCHANGES-454] Deprecate Trac integration

### PR DESCRIPTION
**Pull request by slawekjaranowski**
https://issues.apache.org/jira/browse/MCHANGES-454

### CODE REVIEW

File: src/main/java/org/apache/maven/plugins/changes/announcement/AnnouncementMojo.java [hunk 1]
```diff
@@ -477,6 +477,7 @@ public void execute() throws MojoExecutionException {
             }
 
             if (issueManagementSystems.contains(TRAC)) {
+                getLog().warn("TRACT integration is prepared for remove in next major version, lack of maintainers");
```
Comment (elharo) [inline src/main/java/org/apache/maven/plugins/changes/announcement/AnnouncementMojo.java:480]: TRAC, TRACT, or Track?

Comment (slawekjaranowski) [reply src/main/java/org/apache/maven/plugins/changes/announcement/AnnouncementMojo.java]: should be `TRAC` 😄 fixed

Comment (michael-o) [reply src/main/java/org/apache/maven/plugins/changes/announcement/AnnouncementMojo.java]: Tractor 🚜

Comment (pzygielo) [reply src/main/java/org/apache/maven/plugins/changes/announcement/AnnouncementMojo.java]: Please fix the PR title as well.

Comment (slawekjaranowski) [reply src/main/java/org/apache/maven/plugins/changes/announcement/AnnouncementMojo.java]: summary - jira, pr and commit also fixed 😄

Comment (michael-o) [reply src/main/java/org/apache/maven/plugins/changes/announcement/AnnouncementMojo.java]: It is Trac

Comment (slawekjaranowski) [reply src/main/java/org/apache/maven/plugins/changes/announcement/AnnouncementMojo.java]: I like a code review even with simple task ...

File: src/main/java/org/apache/maven/plugins/changes/announcement/AnnouncementMojo.java [hunk 2]
```diff
@@ -477,6 +477,7 @@ public void execute() throws MojoExecutionException {
             }
 
             if (issueManagementSystems.contains(TRAC)) {
+                getLog().warn("Trac integration is prepared for remove in next major version, lack of maintainers");
```
Comment (elharo) [inline src/main/java/org/apache/maven/plugins/changes/announcement/AnnouncementMojo.java:480]: remove --> removal

File: src/site/apt/index.apt.vm [hunk 3]
```diff
@@ -105,6 +105,10 @@ ${project.name}
 | trac-report -> trac-changes     | trac-report.html -> trac-changes.html     |
 *---------------------------------+-------------------------------------------+
 
+** Deprecate Trac integration
+
+  <<Trac>> integration is prepared for remove in next major version, lack of maintainers.
```
Comment (elharo) [inline src/site/apt/index.apt.vm:110]: removal
version due to lack

Comment (slawekjaranowski) [reply src/site/apt/index.apt.vm]: @elharo last suggestions fixed

**Review by elharo — APPROVED** [review]

**Review by pzygielo — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #245

**Comment by jira-importer** [comment]
Resolve #385

**Pull request merged**